projects/plugins/jetpack/sal/class.json-api-links.php

```diff
@@ -249,17 +404,17 @@ protected function get_endpoint_path_versions() {
 
 		foreach ( $this->api->endpoints as $key => $endpoint_objects ) {
 
-			// The key contains a serialized path, min_version and max_version
-			list( $path, $min_version, $max_version ) = unserialize( $key );
+			// The key contains a serialized path, min_version and max_version.
+			list( $path, $min_version, $max_version ) = unserialize( $key );         // phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.serialize_unserialize -- Legacy, see serialization at See serialisation at class.json-api.php.
```

Comment:
See class.json-api.php - we serialize data here and a comment was added with the following:
`// phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.serialize_serialize -- Legacy, possibly depended on elsewhere.`.
Based on that I've similarly ignored the PHPCS error for the use of the unserialize function here. However this may be worth a @todo for the future?

Reply:
@anomiex do you think a `@todo` would be a good idea here?

Reply:
Wouldn't hurt, although I think there's already one in class.json-api.php.

Reply:
Good point :D I'll add it in anyway to make it even more obvious.